TextMining.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import pickle
+import string
+import random
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+#data file downloaded once, text has been saved through pickle
+#text request needs to be be re-pickled when a new user runs the analysis(uncommet from line 10-16 and then run the file, then comment that part again after the text is pickled to user's computer)
+
+# the_adventure_of_sherlock_holmes = requests.get('http://www.gutenberg.org/ebooks/1661.txt.utf-8').text
+# line_split=the_adventure_of_sherlock_holmes.split('\n')
+# modified_sherlock_text=line_split[57:12682]
+#
+# f = open ('sherlock_texts.pickle', 'wb')
+# pickle.dump(modified_sherlock_text,f)
+# f.close()
+
+
+input_file = open('sherlock_texts.pickle', 'rb')
+reloaded_copy_of_texts = pickle.load(input_file) #reload the pickled file
+
+sherlock_texts = open('sherlock.txt','w') #open a writable text and convert the pickled list to the string
+sherlock_texts.write ('\n'.join (reloaded_copy_of_texts))
+sherlock_texts.close()
+
+def process_line(line, hist):
+    line = line.replace('-', ' ')
```

Comment:
please add docstrings for all the functions you write. You can take a look at the google style used here: https://github.com/sd17fall/GeneFinder/blob/formatted/gene_finder.py. 

I can't not figure out what this function is doing unless I go read every line. When the code gets longer, having no docstrings is really frustrating to the reviewer.